test/parallel/test-stringbytes-external-exceed-max-by-1.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+// v8 fails silently if string length > v8::String::kMaxLength
+// v8::String::kMaxLength defined in v8.h
+const kStringMaxLength = process.binding('buffer').kStringMaxLength;
+
+try {
+  new Buffer(kStringMaxLength * 3);
+} catch(e) {
+  assert.equal(e.message, 'Invalid array buffer length');
```

Comment:
@Trott Can you change this? The error message has changed recently due to a difference in Buffer allocation.